### ISSUE TITLE
Editor toolbar not accessible by keyboard 

### ISSUE THREAD

**Issue by mcgratay** [pressbooks/pressbooks#1646]
### Description

Visual and text toolbars are not accessible by keyboard. When a user tabs through options on the following pages, the editor tools are skipped completely: 

- Chapter/Part/Front Matter/Back Matter editors
- Book Information Long Description and Copyright Notice editors
- Cover Generator About editor
![Screen Shot 2019-04-04 at 1 59 40 PM](https://user-images.githubusercontent.com/39064028/55577592-ed471600-56e1-11e9-8efc-4b6b10303f36.png)
![Screen Shot 2019-04-04 at 1 59 46 PM](https://user-images.githubusercontent.com/39064028/55577593-ed471600-56e1-11e9-93f4-a1c9a123e150.png)

**Comment by SteelWagstaff** [comment]
TinyMCE's documentation: https://www.tiny.cloud/docs/advanced/accessibility/ + https://www.tiny.cloud/docs-3x/extras/TinyMCE3x@Accessibility/ indicate that users can jump into these TinyMCE controls, but as this does appear to require further action from a user than the expected 'tabbing' through, I am investigating further.
